vipps-recurring-api.md

```diff
@@ -537,36 +566,110 @@ when the product is shipped.
 
 ### Campaigns
 
-A campaign in recurring is a period where the price is lower than usual, and
-this is communicated to the customer with the original price shown for comparison.
+Campaigns can be used to lower the price of the agreement for a limited amount of time.
+Recurring supports 4 different campaign types: price campaign, period campaign, event campaign, and full flex campaign. See more about the different campaign types in the table below. This is communicated to the customer with the original price shown for comparison.
 Campaigns can not be used in combination with variable amount, see more about variable amount [here](#Recurring-agreements-with-variable-amount).
 
-![Campaign example](images/CampaignExample.PNG)
+| Campaign types        | Description                                                                                              | Example                                                                                 |
+|-----------------------|----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| `price campaign`      | A set price until a set date. Same interval as agreement.                                                | 10kr per 1 week interval until 2022-05-01T00:00:00Z and then 200kr per 1 week interval  |
```

Comment:
```suggestion
| `price campaign`      | Different interval price until specified date. Same interval as agreement.                                                | 10kr per 1 week interval until 2022-05-01T00:00:00Z and then 200kr per 1 week interval  |
```

A lot of people will have trouble understanding the difference between "A set price until a set date" and "A set price for a given duration". I suggest being a bit more specific so they might understand without needing the example.